app/models/concerns/foreman_openscap/compliance_status_scoped_search.rb

```diff
@@ -109,11 +109,15 @@ def apply_condition(scope, negate, conditions)
 
     included do
       if ForemanOpenscap.with_katello?
-        has_one :lifecycle_environment, :through => :host
+        has_one :content_facet, :through => :host
+        has_many :content_view_environment_content_facets, through: :content_facet, class_name: 'Katello::ContentViewEnvironmentContentFacet'
+        has_many :content_view_environments, through: :content_view_environment_content_facets
+        has_many :content_views, through: :content_view_environments
```

Comment:
I thought you'll fix that in the way it's done for the other searches through `ext_method`, but that's seems fine as well :)